content/updates/2020-09/1-flat-files-exports-with-labels.md

```diff
@@ -0,0 +1,32 @@
+---
+pim_announcement_img: ./img/Quick-export-new-UI_Export-with-labels.png
+pim_announcement_alt_img: "Flat files exports with labels"
+pim_announcement_audience:
+- CE
+- EE
+---
+
+# Export flat files with labels
+::: meta-data type="New" features="Flat files exports" available="early September" link-to-doc="../articles/exports.html"
+:::
+
+For your return from the summer holidays, we decided to tackle a widely shared issue, the impossibility for humans to read your flat files exports.
+
+Until then, it was nearly impossible for your clients to be able to decipher the codes into your product exports, and you had to re-work every file to be human-friendly.
+
+So we decided to implement an improvement into:
+* Quick export
+* Export profile
+
+Now, a new option named `Export with label` is available, helping you to display your product export with labels, translated into the selected locale (that must have been activated in your PIM).
+
+This way, your product exports will have their headers and their values translated, making it easy to share to your clients.
+# New UI for Quick Export
```

Comment:
You can't have two titles in one announcement. If this is a different one, please create a separate md file. If the two features are too close and totally linked, you'll have to use sub-sections.

Reply:
Here, I think it can be a whole new dedicated announcement.